tips/TIP-0022/tip-0022.md

```diff
@@ -29,15 +29,15 @@ Various other protocol TIPs rely on certain constants that need to be defined fo
 | Min PoW Score                         | 1500.0           | Minimum PoW score for blocks to pass syntactic validation.                                                                                                       |
 | First Milestone Index                 | 1                | First valid milestone index.                                                                                                                                     |
 |                                       |                  |                                                                                                                                                                  |
-| Max IOTA Supply                       | 4600000000000000 | Total amount of IOTA coins in circulation.                                                                                                                       |
+| Max IOTA Supply                       | 4600000000000000 | Total amount of micro (1/1000000 IOTA) coins                                                                                                                       |
```

Comment:
removed the in circulation, as through the vesting mechanic it won't in circulation for a while